File: Leetcode/cpp/1637.cpp
class Solution {
public:
    int maxWidthOfVerticalArea(vector<vector<int>>& points) {
        std::sort(points.begin(), points.end());
        
        int output = 0;
        for (int i = 1; i < points.size(); i++) {
            output = std::max(output, points[i][0] - points[i - 1][0]);
        }
        return output;
    }
};

File: Leetcode/Python/_1637.py
class Solution:
    def maxWidthOfVerticalArea(self, points: List[List[int]]) -> int:
        points.sort()
        output = 0
        for i in range(1, len(points)):
            output = max(output, points[i][0] - points[i - 1][0])
        return output
